lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -837,7 +840,22 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
       RETURN_CHILD(m_is_escalated_sp, isEscalated, bool_type);
     case 10:
       RETURN_CHILD(m_is_enqueued_sp, isEnqueued, bool_type);
-    case 11:
+    case 11: {
+      if (!m_child_tasks_sp) {
+        const auto &tasks = m_task_info.childTasks;
+        std::string mangled_typename =
+            mangledTypenameForTasksTuple(tasks.size());
+        CompilerType tasks_tuple_type =
+            m_ts->GetTypeFromMangledTypename(ConstString(mangled_typename));
+        DataExtractor data{tasks.data(), tasks.size() * sizeof(tasks[0]),
+                           endian::InlHostByteOrder(), sizeof(void *)};
```

Comment:
Maybe use multiples of the pointer size on the target, since we still need to bake in this knowledge.

Reply:
Same comment, may have been my misunderstanding!